BREEZE.rst

```diff
@@ -265,6 +265,40 @@ You should set up the autocomplete option automatically by running:
 
 You get the auto-completion working when you re-enter the shell.
 
+Breeze on Windows
```

Comment:
One comment - I think for now we should develop a "parallel documentation. I have not thought about it too much - but I believe for now - until we open Breeze2 for others, we should keep the docs in `./dev/breeze/doc`. 

I am not 100% sure what will be the best structure there. Should we have a separate "Breeze_install.md" ? Or everything in `README.md` or something else? Maybe as a 'fresh' user - you could come up with some proposal @edithturn ? . For sure we should not add it to the current Breeze docuementation.

Reply:
And we should also add docs for other platforms. Actually `pipx` is not Windows specific. If as @uranusjr says "forward slash" works on Windows. then we could provide universal installation instructions for any OS.

Reply:
I agree with this, BREEZE.rst documentation is already big. I don't know about the structure yet because I am not familiar with everything, but next week I came come with an idea. For now ./dev/breeze is good.

Reply:
If I understand correctly all the documentation related to Breeze2 should be in a separate directory, right? If it is like that, I think this directory is good:  `./dev/breeze/doc`, but we should think about what things we will behave here, For now, I just know we have BREEZE.rst documentation. (Correct me if I am wrong)